src/actions/operator.ts

```diff
@@ -304,11 +304,6 @@ export class YankOperator extends BaseOperator {
 
     Register.put(text, vimState, this.multicursorIndex);
 
-    if (vimState.currentMode === Mode.Visual || vimState.currentMode === Mode.VisualLine) {
-      vimState.historyTracker.addMark(start, '<');
-      vimState.historyTracker.addMark(end, '>');
-    }
-
     await vimState.setCurrentMode(Mode.Normal);
```

Comment:
This is probably old code. lastVisualSelection is tracked in vimState and motions must access this, we dont need/sould delete this.